src/XRP/network-clients/json-rpc-network-client.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios'
+
+interface JsonRequest {
+  method: string
```

Comment:
is this method different from REST methods? ie, is this an `rpcMethod` or some such?

Reply:
yeah, this field is where you specify which rpc command you're invoking.  it needs to be called `method` because that's what rippled is expecting